packages/api/src/webhooks/cannon.ts

```diff
@@ -497,11 +494,11 @@ export default class WebhookCannon {
         responseBody.substring(0, 1024)
       ).toString("base64");
 
-      await this.db.webhookResponse.create({
```

Comment:
Should we remove this field from the class and fix all usages?

Reply:
I think I'll save that for a separate PR 👍